web/controllers/unit_controller.ex

```diff
@@ -21,7 +22,8 @@ defmodule OrgtoolDb.UnitController do
 
     case Repo.insert(changeset) do
       {:ok, unit} ->
-        unit = unit |> Repo.preload([:unit_type])
+        unit = unit
+         |> Repo.preload([:unit_type, :unit, :units, :members, :leaders, :applicants])
```

Comment:
👍 those were missing but best put them at the top in`@preload` and `@opts` like we have on the other controllers to have unified way of doing it.